pkg/apis/kafka/v1beta1/kafka_types.go

```diff
@@ -0,0 +1,56 @@
+package v1beta1
```

Comment:
Is it not possible to import this from the strimzi project?

Reply:
No, they are a Java operator and do not generate Go artifacts.